app/src/Test/TesterFactory/CompiledTester.ts

```diff
@@ -121,7 +122,13 @@ export default class CompiledTester extends Tester {
   getDefaultExecutableFileName(debug: boolean): string {
     let defaultName = Util.replaceAll(Path.parse(this.filePath).name, " ", "");
     if (debug) defaultName += "debug";
-    defaultName += ".exe";
+    let fileExtension = this.config.executableFileExtension;
+    if (fileExtension) {
+      // replace for cases where user includes unnecessary period
+      // ensures the option still works as expected
+      fileExtension = fileExtension.replace(/[^A-Za-z]/g, '');
```

Comment:
Oh yeah, that's right. Thanks for catching that. I originally used that regex to catch whitespaces as well but forgot that it would remove a lot more.